bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -310,14 +310,14 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
           {{ email_newsletter_form() }}
         </section>
       </div>
-      <div id="mosaic-group-1" class="ri-grid" data-loading="{{ static('img/mosaic/loading.gif') }}">
+      <div id="mosaic-group-1" class="ri-grid" data-loading="{{ static('img/about/manifesto/mosaic/loading.gif') }}">
```

Comment:
This img is 404 on the manifesto page. I think it should be `img/mozorg/about/manifesto/mosaic/loading.gif`